src/operator/dropout.cc

```diff
@@ -31,15 +31,35 @@ Operator *DropoutProp::CreateOperatorEx(Context ctx, std::vector<TShape> *in_sha
 DMLC_REGISTER_PARAMETER(DropoutParam);
 
 MXNET_REGISTER_OP_PROPERTY(Dropout, DropoutProp)
-.describe(R"(Applies dropout to input.
-During training, each element of the input is randomly set to zero with probability p.
-And then the whole tensor is rescaled by 1/(1-p) to keep the expectation the same as
-before applying dropout. During the test time, this behaves as an identity map.
-)")
-.add_argument("data", "NDArray-or-Symbol", "Input data to dropout.")
+.describe(R"(Applies dropout operation to input array.
+
+- During training, each element of the input is randomly set to zero with probability p.
+  The whole array is rescaled by :math:`1/(1-p)` to keep the expectated sum of the input unchanged.
```

Comment:
"expectated" -> "expected"  (expectated not a word in English although it apparently was a word in Latin :P)